content/blog/2023/11/2023-11-10-Jenkins-Contributor-Summit-in-Brussels.adoc

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "Jenkins Contributor Summit in Brussels"
+tags:
+- events
+- community
+- contribute
+author: jmMeessen
+opengraph:
+  image: /images/post-images/2023/11/ContribSummit/Contributor_summit_2024.png
+---
+
+
+We are happy to announce that we will hold a **Jenkins Contributor Summit** in Brussels prior to FOSDEM. 
+
+image:/images/post-images/2023/11/ContribSummit/Jenkins_desk.jpg[Contributor Summit Desk,float=center]
+
+image:/images/post-images/2023/11/ContribSummit/fosdem.videobox.logo.svg[image,width=200,float=right,role=center]
+
+link:https://fosdem.org/2024/[FOSDEM] is a free event for software developers to meet, share ideas and collaborate. 
+Every year, thousands of developers of free and open source software from all over the world gather at the event in Brussels. 
+It takes place on Saturday 3 & Sunday 4 February 2024.
+
+
+== What is a Jenkins Contributor Summit?
+
+image:/images/post-images/2023/11/ContribSummit/image.jpg[image,width=300,float=right,role=center]
+
+The Jenkins Contributor Summit brings together current and future contributors to the Jenkins project. 
+At this event  we will talk about the current state of the project and its future evolution. 
+It is a great opportunity to meet "in real life" fellow actors of the Jenkins community.
+
+It is a day-long event (from 10h00 to 17h00) featuring presentations on the "State of Jenkins", Project/SIG updates, discussion on various key projects, "Ignite Talks", and demos.
+
+Don't hesitate to reach out (see contact section) if you want to submit a presentation, a talk, or a demo.
+
+
+== Key Dates:
+* **Jenkins Contributor Summit:** Friday 2 February 2024
+* **FOSDEM:** Saturday 3 and Sunday 4 February 2024
+
+image:/images/post-images/2023/11/ContribSummit/Jenkins_at_fosdem.jpg[Figure 1,title="test title"]
+The Jenkins booth at FOSDEM
+
+== Location:
+* _Betacowork,_ located at _Rue des Pères Blancs 4, 1040 Brussels, Belgium_
+(Easily accessible from downtown Brussels or the FOSDEM venue.)
+
+== Entry Fee:
+* No cost to attend.
+* Registration will be required (room capacity: 30). 
+* Event is in-person only.
+* No catering is foreseen but food can be purchased in the neighborhood.
```

Comment:
```suggestion
* There is no cost to attend.
* Registration will be required because there is a 30 person room capacity. 
* The FOSDEM event is in-person only.
* No catering is foreseen, but food can be purchased in the neighborhood.
```
Suggestion to add some syntax for grammar & punctuation

Reply:
It is not the FOSDEM event that is in person but the contributor summit